dev/com.ibm.ws.security.oauth/test/com/ibm/ws/security/oauth20/web/OAuth20EndpointServicesTest.java

```diff
@@ -183,6 +183,19 @@ public void testIntrospectExpired() {
         }
     }
 
+    @Test
+    public void testURLEncodeParams() {
+        OAuth20EndpointServices oas = new OAuth20EndpointServices();
+        String in = "https://www.somewhere.com?item=s巧";
+        String expected = "https://www.somewhere.com?item%3Ds%E5%B7%A7";
```

Comment:
I don't think is actually what we want - the `=` characters shouldn't get getting encoded unless they're actually part of a parameter name or value. You should be able to do an experiment to see the difference in behavior by going to these pages:
- https://github.com/OpenLiberty/open-liberty/pull/9706/files?w=1 (Loads the diff ignoring white space)
- https://github.com/OpenLiberty/open-liberty/pull/9706/files?w%3D1 (The encoded `=` character is considered part of the parameter name, so the browser will actually just end up dropping that parameter entirely and will *not* load the diff ignoring white space)

Reply:
fixed, thanks